GOVERNANCE.md

```diff
@@ -156,7 +180,8 @@ To nominate a new Collaborator:
    block, add a comment in the private discussion stating you're planning on
    opening a public issue, e.g. "I see a number of approvals and no block, I'll
    be opening a public nomination issue if I don't hear any objections in the
-   next 72 hours".
+   next 72 hours". _Ideally_ this should only be done after any questions,
+   concerns, objects, etc have been resolved.
```

Comment:
For the same reason navigation apps will tell you "After 500 meters, take a right" and not "Take a right after 500 meters", we should change the order here (otherwise, folks might first take the action before reading that they shouldn't have). So it should be something like "…if the nomination proposal has received some support and no explicit block, and any questions/concerns have been addressed, add a comment…"

Reply:
To make sure the edit is correct can I ask you to possibly use the suggestion mechanism.

Reply:
It's too far up for me to do that unfortunately, GH won't let me